Tella/Scenes/HomeView/ViewModel/FileListViewModel.swift

```diff
@@ -367,6 +367,23 @@ extension FileListViewModel {
         }
     }
     
+    
+    func checkReportsBeforeDelete() -> Bool {
```

Comment:
what do you think if we change it to this ?

    func filesAreUsedInConnections() -> Bool {
        guard let database = appModel.tellaData?.database else { return false }
        let fileIds = selectedFiles.compactMap { $0.id }
        return database.checkFilesInReports(ids: fileIds)
    }